Userland/Libraries/LibGfx/Bitmap.h

```diff
@@ -115,18 +115,7 @@ class Bitmap : public RefCounted<Bitmap> {
 
     [[nodiscard]] size_t pitch() const { return m_pitch; }
 
-    [[nodiscard]] static unsigned bpp_for_format(BitmapFormat format)
-    {
-        switch (format) {
-        case BitmapFormat::BGRx8888:
-        case BitmapFormat::BGRA8888:
-            return 32;
-        default:
-            VERIFY_NOT_REACHED();
```

Comment:
this was neat _ersatz_ for pattern matching, probably worth keeping in the new location.

Reply:
The current code ensures that all values are validated at compile-time, which provides a significant advantage: it prevents the possibility of introducing a new color space without explicitly adding support for it in this function. This approach eliminates the risk of runtime crashes due to unhandled cases, as any missing implementation will result in a compile-time error rather than a runtime failure.

Reply:
Does any caller call this  method with untrusted user input? Could any caller be reasonably created to call with untrusted user input? And by untrusted user input I am including image data from an undecided image, as well as information that comes from IPC.

Reply:
In this case, since it `return`s, the `VERIFY_NOT_REACHED()` still should be added - just after the switch.